crates/optimism/tests/Makefile

```diff
@@ -81,7 +81,7 @@ test-e2e-sysgo: unzip-contract-artifacts build-contracts
 	export DISABLE_OP_E2E_LEGACY=true; \
 	export DEVSTACK_ORCHESTRATOR=sysgo; \
 	export OP_RETH_ENABLE_PROOF_HISTORY=true; \
-	export OP_RETH_EXEC_PATH="${SOURCE_DIR}/../../../target/release/op-reth"; \
+	export OP_RETH_EXEC_PATH="${SOURCE_DIR}/../../../target/debug/op-reth"; \
```

Comment:
checkout the hive test profile
https://github.com/op-rs/op-reth/blob/f7603f721a72b90b96d562d9016eb8e4546bf8f4/Cargo.toml#L304-L308